### PR TITLE
Replace Arguments Map for parse-function

### PR DESCRIPTION
**Pull request by gunar**

### CODE REVIEW

**Comment by keenahn** [comment]
Squash your commits :) But looks good.

Should we lock the major version of node?

File: index.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const getMessageIndex = fn => {
 
 function avowify(fn, methodName) {
   return function (...args) {
-    try { fn(...args) }
+    try { return fn(...args) }
```
Comment (keenahn) [inline index.js]: 👍

**Comment by gunar** [comment]
Squashed. I don't think we need to lock major version of node.

**Comment by gunar** [comment]
Auto published, woohoo!

image

**Pull request merged**